docs/content/users/extend/customizing-images.md

```diff
@@ -46,7 +46,7 @@ Examine the resultant generated Dockerfile (which you will never edit directly),
 
 Examples of possible Dockerfiles are `.ddev/web-build/Dockerfile.example` and `.ddev/db-build/Dockerfile.example`, created in your project when you run [`ddev config`](../usage/commands.md#config).
 
-You can use the `.ddev/*-build` directory as the Docker “context” directory as well. So for example, if a file named `README.txt` exists in `.ddev/web-build`, you can use `ADD README.txt /` in the Dockerfile.
+You can use the `.ddev/*-build` directory as the Docker “context” directory as well. So for example, if a file named `README.txt` exists in `.ddev/web-build`, you can use `ADD .ddev/web-build/README.txt /` in the Dockerfile.
```

Comment:
Apply the changes from v1.21.5.

Reply:
```suggestion
You can use the `.ddev/*-build` directory as the Docker “context” directory as well. So for example, if a file named `README.txt` exists in `.ddev/web-build`, you can use `ADD README.txt /` in the Dockerfile.
```

Let's skip this one as that feature is being reverted, so this will just conflict with https://github.com/ddev/ddev/pull/4728